core/tauri/src/ipc/authority.rs

```diff
@@ -64,15 +65,15 @@ impl Origin {
 }
 
 impl RuntimeAuthority {
-  pub(crate) fn new(resolved_acl: Resolved) -> Self {
+  #[doc(hidden)]
```

Comment:
do we need to hide this?

Reply:
I don't want anyone to use this API (though they shouldn't.. it's useless without attaching to a manager - which you can't - and it's only used during codegen)

Reply:
Ah, nvm, I thought it had more methods that users could use for their own ACL implementation or something.